port/zos390/omrintrospect.h

```diff
@@ -34,22 +34,26 @@
 #include "omrintrospect_common.h"
 #include "omrutil.h"
 
-#pragma linkage(getthent, OS_UPSTACK)
 #if defined(OMR_ENV_DATA64)
+#pragma linkage(getthent, OS)
 #pragma map(getthent, "BPX4GTH")
 #else
+#pragma linkage(getthent, OS_UPSTACK)
 #pragma map(getthent, "BPX1GTH")
 #endif
 
-#pragma linkage(pthread_quiesce, OS_UPSTACK)
 #if defined(OMR_ENV_DATA64)
 #pragma map(pthread_quiesce, "BPX4PTQ")
 #else
+#pragma linkage(pthread_quiesce, OS_UPSTACK)
 #pragma map(pthread_quiesce, "BPX1PTQ")
 #endif
 
+#if defined(OMR_ENV_DATA64)
+#else
```

Comment:
Just say `#if !defined(OMR_ENV_DATA64)` and add the missing `#endif` comment.

Reply:
looks like I have your feedback in the other PR: https://github.com/eclipse/omr/pull/7321
Might just remove this file from this PR, as that one seems to handle more updated/latest code for `omrintrospect.h`